tests/integration/test_render.py

```diff
@@ -957,7 +957,7 @@ def update_output(value):
         self.wait_for_text_to_equal('#output-post', 'request_post changed this text!')
         self.wait_for_text_to_equal('#output-post-payload', '{"output":"output-1.children","changedPropIds":["input.value"],"inputs":[{"id":"input","property":"value","value":"fire request hooks"}]}')
         self.wait_for_text_to_equal('#output-post-response', '{"props":{"children":"fire request hooks"}}')
-        self.percy_snapshot(name='request-hooks')
+        self.percy_snapshot(name='request-hooks render')
```

Comment:
oh interesting - was one of these snapshots blocking the other one previously? How did you discover this? Do we need some additional mechanism (like a list of previously-used names inside `percy_snapshot` - though I guess in this case that wouldn't help as one's the new framework and the other is still using the old system) to ensure we don't double up a name again?

Reply:
And I guess the next question: do these two tests actually both have distinct value, or should we delete one of them?

Reply:
I got a 400 error when the test job start to have parallelism > 1

Reply:
oh weird - so it allows multiple snapshots with the same name from the same process but not from different processes? very strange, though useful I guess as any test we do in our own code could only catch duplicates in the same process.

Reply:
there was another issue with the previous approach, as we spin up a browser per test vs per session, the percy_finalize was called after each test case. 

anyway, there are some extra integration under the hood from percy client, it adds some extra protection for all major CI tools.

Reply:
Ah, got it - right, it's probably the extra finalizing that buried the duplicate, and now percy will be taking care of ensuring uniqueness.